prefs.js

```diff
@@ -31,6 +31,32 @@ const TimezoneExtensionPrefsWidget = new GObject.Class({
                                  use_markup: true, margin_bottom: 6,
                                  hexpand: true, halign: Gtk.Align.START}));
 
+        let box = new Gtk.Box({orientation: Gtk.Orientation.HORIZONTAL, spacing: 5});
+
+        this._check = new Gtk.CheckButton();
+        this._check.connect("toggled", Lang.bind (this, this._toggledCB));
+
+        this._mlabel = new Gtk.Label({ label: '<b>Highlight working hours: From</b>',
+                                 use_markup: true, margin_bottom: 6,
+                                 hexpand: false, halign: Gtk.Align.START });
+
+        this._hour1 = Gtk.SpinButton.new_with_range (0, 23, 1);
+        this._hour2 = Gtk.SpinButton.new_with_range (0, 23, 1);
```

Comment:
I use Ubuntu (not a real GNOME). So, I can't say how date/time is handled for a real GNOME user. Anyway, it was tested.

![image](https://cloud.githubusercontent.com/assets/1702923/24082831/8851b242-0caa-11e7-997b-0048ae700348.png)